test/nls/feasibility-form-nls.jl

```diff
@@ -4,57 +4,59 @@
     JF(x) = Float64[0 0 1 0; 0 0 0 1]
     HF(x, w) = zeros(4, 4)
 
-    nls = FeasibilityFormNLS(SimpleNLSModel())
-    n = nls.meta.nvar
-    m = nls.meta.ncon
-    ne = nls_meta(nls).nequ
+    for dt in [Float64, Float32]
+      nls = FeasibilityFormNLS(SimpleNLSModel(zero(dt)))
```

Comment:
```suggestion
    for T in [Float64, Float32]
      nls = FeasibilityFormNLS(SimpleNLSModel(T))
```
Again, you can plug into `@testset`.

Reply:
Thanks for all the comments! I am not sure what you meant here. Do you suggest this:
```
@testset "NLS API" begin for T in [Float64, Float32]
...
end end
```
or
```
for T in [Float64, Float32]
  @testset "NLS API $T" begin 
  ...
  end 
end
```
?